baybe/simulation/lookup.py

```diff
@@ -159,3 +139,41 @@ def _look_up_targets_from_dataframe(
 
     # Add the lookup values
     queries.loc[:, target_names] = np.asarray(all_match_vals)
+
+
+def label_columns(
```

Comment:
can we come up wit a better name that also expresses this is a wrapper or an object that does something and not just a variabels toring the columns?

Reply:
Maybe `evolve_ndarray_callable` or similar?

Reply:
> Maybe `evolve_ndarray_callable` or similar?

For my taste i) too unspecific what evolve means here and ii) needs to work also with tensors.

How about the version I've just pushed, where I now also replaced all legacy calls to `botorch_function_wrapper`? Called the utility `label_input_and_output_columns`. A longer name would have been `label_callable_input_and_output_columns`, but because we mention input and outputs, the `callable` in the name is actually redundant. Thoughts?

Reply:
im having my issues with the word `label` since this does not just labels something but converts the function etc, so why not `array_lookup_to_df` or something with `array2df` / `array2df_wrapper`

Reply:
renamed it to `arrays_to_dataframes` 👍🏼